### DISCUSSION TITLE
running a terminal in theia installed on a docker container

### DISCUSSION THREAD

**Discussion by ctamietto** [eclipse-theia/theia#13808] [Q&A]
I have build the docker container cloning the repository
 
**https://github.com/eclipse-theia/theia-blueprint.git**

and  then build the image with the command :

**docker build -t theia-ide -f browser.Dockerfile .**

then I run the container 

**docker run -p=3000:3000 --rm theia-ide**

access the ide with the url :

http://localhost:3000/#/home/project

at that point I have opened a terminal :

the terminal opened without a prompt 

![image](https://github.com/eclipse-theia/theia/assets/3806889/1d670351-8918-4524-a459-6b01ad132f11)

and the cpu go to 100% 

![image](https://github.com/eclipse-theia/theia/assets/3806889/2bc30b09-f40b-4733-9dd6-b5c08fc31523)

for some minutes 
then finally the terminal is working .

my question is 

am I doing something wrong ?

is normal that running a terminal take so much time and resource consuming 

thanks for any answer

**Comment by Dinesh-4320** [comment]
Do you also have the issue in editing files during this time?
